purchase_partner_default_journal/i18n/purchase_partner_default_journal.pot

```diff
@@ -0,0 +1,42 @@
+# Translation of Odoo Server.
```

Comment:
we don't really use this any more because transifex replaces it, so no need to submit